packages/identity/src/identity/secp256k1.test.ts

```diff
@@ -0,0 +1,117 @@
+import { DerEncodedPublicKey } from '@dfinity/agent';
+import { randomBytes } from 'crypto';
+import { sha256 } from 'js-sha256';
+import Secp256k1 from 'secp256k1';
+import { Secp256k1KeyIdentity, Secp256k1PublicKey } from './secp256k1';
+
+function fromHexString(hexString: string): ArrayBuffer {
+  return new Uint8Array((hexString.match(/.{1,2}/g) ?? []).map(byte => parseInt(byte, 16))).buffer;
+}
+
+// DER KEY SECP256K1 PREFIX = 3056301006072a8648ce3d020106052b8104000a03420004
+const testVectors: Array<[string, string]> = [
```

Comment:
Would it make sense to add a comment on what the vectors contain and how / where they were obtained?

Reply:
Yes please! Could you suggest a comment?

Reply:
Sure, I assume the keys were generated as I suggested [here](https://github.com/Psychedelic/plug-controller/pull/33#issuecomment-895121742) using OpenSSL.

So the comment could be: 
```
// These test vectors contain the hex encoding of the corresponding raw and DER versions 
// of secp256k1 keys that were generated using OpenSSL as follows: 
// openssl ecparam -name secp256k1 -genkey -noout -out private.ec.key
// openssl ec -in private.ec.key -pubout -outform DER -out ecpubkey.der
// hexdump -ve '1/1 "%.2x"' ecpubkey.der
```